qiskit/transpiler/passes/basis/unroll_3q_or_more.py

```diff
@@ -63,6 +64,15 @@ def run(self, dag):
             elif self.basis_gates is not None and node.name in self.basis_gates:
                 continue
 
+            if isinstance(node.op, ControlFlowOp):
+                unrolled_blocks = []
+                for block in node.op.blocks:
+                    dag_block = circuit_to_dag(block)
+                    unrolled_dag_block = self.run(dag_block)
+                    unrolled_circ_block = dag_to_circuit(unrolled_dag_block)
+                    unrolled_blocks.append(unrolled_circ_block)
+                node.op = node.op.replace_blocks(unrolled_blocks)
```

Comment:
Same question from the other PR about whether we have a utility function for this, to do things like not update when there's no updates to the blocks.